app/controllers/ui_job_wizard_controller.rb

```diff
@@ -24,6 +24,19 @@ def map_template_inputs(template_inputs_with_foreign)
     template_inputs_with_foreign.map { |input| input.attributes.merge({:resource_type_tableize => input.resource_type&.tableize }) }
   end
 
+  def default_category
+    default_template&.job_category
+  end
+
+  def default_template
+    setting_value = Setting['remote_execution_form_job_template']
+    return nil unless setting_value
+
+    template = JobTemplate.authorized(:view_job_templates).find_by :name => setting_value
+    return nil unless template
+    template
+  end
```

Comment:
All in all we could even do something like this, but that boils down to personal preference. Either way is fine as long we don't do unneeded checks

```suggestion
  def default_template
    if setting_value = Setting['remote_execution_form_job_template']
      JobTemplate.authorized(:view_job_templates).find_by :name => setting_value
    end
  end
```

Reply:
Used this instead, with added parenthesis to make rubocop happy.
Thanks for the review!